game-core/src/main/java/games/strategy/engine/framework/ui/DefaultGameChooserEntry.java

```diff
@@ -4,126 +4,67 @@
 import games.strategy.engine.data.GameData;
 import games.strategy.engine.data.GameParseException;
 import games.strategy.engine.data.gameparser.GameParser;
-import games.strategy.triplea.Constants;
+import games.strategy.engine.data.gameparser.ShallowGameParser;
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
-import java.util.Objects;
 import java.util.Optional;
 import java.util.logging.Level;
-import javax.annotation.Nonnull;
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
 import lombok.extern.java.Log;
 import org.triplea.java.UrlStreams;
 
 @Log
-final class DefaultGameChooserEntry implements GameChooserEntry {
-  private final URI url;
-  @Nonnull private GameData gameData;
-  private boolean gameDataFullyLoaded = false;
-  private final String gameNameAndMapNameProperty;
+@EqualsAndHashCode(of = "uri")
+@Getter
+public class DefaultGameChooserEntry implements Comparable<DefaultGameChooserEntry> {
```

Comment:
Also a consideration:
Now that `GameChooserEntry` is no longer a thing this class could get renamed

Reply:
Indeed. I've been waiting on the rename for a moment for a few reasons:
- renames are really tough on review as the file often will look new. File renaming and package moves is perhaps the biggest downside of doing reviews, reviews do discourage those operations because the diff gets so messed up
- a rename done on its own would conflict with work that is continuing on from this point. A post-merge really indicates that I needed to incorporate these updates to keep going in a clean manner. The merge conflicts have been working out pretty okay so far, but I have been encountering some even with a pretty quick cadence of merges. So this is to say that there is already future work that would make a rename not nice as it would conflict badly with that, so it's a matter of finding the time to fit in
- hasn't necessarily been that important per-say. The name 'default game chooser entry' is not too horrible necessarily.
- not clear if this class has a long term future or would be refactored away to another form or just reduced down to a map. It is now only a tuple class that holds a string and a URI, really TBD if it'll survive